src/main/java/dataLoader/csvDataLoader.java

```diff
@@ -0,0 +1,33 @@
+package dataLoader;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.io.StringReader;
+
+/**
+ * Simple class that parses data set from a csv file
+ * without knowing the headers of the file
+ * */
+
+public class csvDataLoader {
```

Comment:
Classes in Java, by convention, use CamelCase; so please rename this to CsvDataLoader

Reply:
ok